nature/benchmarks/linear_mapper_benchmark.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Linear Mapper Benchmarks."""
+from timeit import timeit
+import retworkx
+from qiskit_nature.mappers.second_quantization import LinearMapper
+from qiskit_nature.problems.second_quantization.lattice.models import IsingModel
+from qiskit_nature.problems.second_quantization.lattice import Lattice
+
+# pylint: disable=redefined-outer-name, invalid-name, attribute-defined-outside-init
+
+
+class LinearMapperBenchmarks:
+    """Linear Mapper Benchmarks."""
+
+    version = 1
+    timeout = 120.0
+    params = [50, 80, 100]
+    param_names = ["Number of nodes"]
+
+    def setup_cache(self):
+        """setup cache"""
+
+        second_q_ops_list = []
+        edge_arr = list((n**11) % 100 for n in range(100))
```

Comment:
Using `pow(n, 11, 100)` should be faster as it implements the fast modular exponentation algorithm.
But do we need to take such a high exponent here?

Reply:
Speed wouldn't really matter here since the function is a `setup_cache` function. But of course, we can change the exponent to a smaller one.

Reply:
Interestingly, I performed a quick test and the former way is actually faster....

```
>>> timeit.timeit('[(n**11) % 100 for n in range(100)]',number=10000)
0.38454898600002707
>>> timeit.timeit('[pow(n,11,100) for n in range(100)]',number=10000)
0.6776183249999974
```
Let's keep it as is then.

Reply:
Of course I agree that in this context it is unimportant. My comment and the discussion is rather theoretical/educational.